src/js/containers/explorer/detail/DetailContentContainer.jsx

```diff
@@ -445,6 +450,70 @@ export class DetailContentContainer extends React.Component {
         });
     }
 
+    goToUnreported(data, date) {
```

Comment:
See comment on `DetailContent.jsx`. The `date` param can be removed, and `this.state.lastUpdate` can be accessed instead.